src/store/selectors/activities/types.ts

```diff
@@ -94,26 +99,22 @@ export type ActivityContent = {
    * display name in the format of a complete sentence.
    */
   description?: (string | CurrencyValue)[];
-} & (
```

Comment:
This doesn't express how our code actually works, since the `CREATE_MEMBER` operation actually does have a body, but no next in chain. But because of how typescript does union types, it seems it didn't actually throw that `body` was present but `nextInChain` wasn't in the activities selector; bummer.